src/app/login/login.component.ts

```
@@ -115,38 +118,44 @@
~          this.confirmationService.confirm('info', res.errorMessage).subscribe((confirmResponse: any) => {
             if(confirmResponse) {
               this.authService.userlogoutPreviousSession(this.userName).subscribe((userlogoutPreviousSession) => {
                 if (userlogoutPreviousSession.statusCode == '200') {
-              this.authService.login(this.userName, encryptPassword, true).subscribe((userLoggedIn) => {
+              this.authService.login(this.userName, encryptPassword, true,this.enableCaptcha ? this.captchaToken : undefined).subscribe((userLoggedIn) => {
                 if (userLoggedIn.statusCode == '200') {
                 if (userLoggedIn.data.previlegeObj != null && userLoggedIn.data.previlegeObj != undefined && userLoggedIn.data.previlegeObj[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)